lib/arc-login.js

```diff
@@ -92,7 +92,7 @@ function getArcKubeconfig() {
                     stdio: 'ignore'
                 }).unref();
             }
-            console.log('started proxy');
+            console.log('Waiting for 2 minutes for kubeconfig to be merged....');
             yield sleep(120000); //sleeping for 2 minutes to allow kubeconfig to be merged

```

Comment:
Put this number in a constant variable. Also, does it always take 2 min or is this an approximation? And what if the Kubeconfig doesn't get generated within 2 min? Is there any callback that can be used to know that it's been generated instead of waiting?

Reply:
Yeah, I have put it in a constant variable. The 2mins is an approximation. If kubeconfig doesnt get generated within the timeout, `fs.chmodSync(kubeconfigPath, '600');` in the subsequent line will throw an exception and we will catch it. There's no possibility for a callback here as the proxy code is not native to javascript.